### DISCUSSION TITLE
PVE下如何安装QEMU Guest Agent，不然不能正常关机

### DISCUSSION THREAD

**Discussion by Danny-Ken** [kkkgo/PaoPaoGateWay#52] [想法]
PVE下如何安装QEMU Guest Agent，不然不能正常关机

**Comment by kkkgo** [comment]
已增加acpid，把QEMU Guest Agent禁用应该可以正常关机，可以使用docker定制新镜像。如果需要QEMU Guest Agent，可以使用fullmod定制。
